docs/manuals/core/config/scenario-config.md

```diff
@@ -68,11 +68,17 @@ associated to a Cycle corresponding to their creation date. See documentation on
 # Configuring scenario comparators
 
 Let us imagine a common situation where the pipeline configuration `pipeline_config` has been created with `datanode_config` as one of the data node configuration for this the pipeline configuration.
-The function `compare_function` has also been defined:
+The function `compare_function` is also defined as followed:
 
 ```python
 from taipy import Config
 
+def compare_function(*data_node_results):
```

Comment:
What about making it completely explicit by naming the variable `data_node_read_results`? Is it a too long name?

Reply:
Sure! Makes it even clearer 😃